index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Hackathon Resume</title>
+        <link rel="stylesheet" href="style.css">
+        <!-- <script type="text/javascript" src="resume.json"></script> -->
+    </head>
+    <body>
+        <div id="myData"></div>
+        <script>
+            fetch('resume.json')
+                .then(function (response) {
+                    return response.json();
+                })
+                .then(function (data) {
+                    appendData(data);
+                })
+                .catch(function (err) {
+                    console.log('error: ' + err);
+                });
+    
+            function appendData(data) {
+                var mainContainer = document.getElementById("myData");
+                for (var i = 0; i < data.length; i++) {
+                    var div = document.createElement("div");
+                    // div.innerHTML = 'Name: ' + data[i].firstName + ' ' + data[i].lastName;
+                    // mainContainer.appendChild(div);
+                }
```

Comment:
I ran into the same problem. Manipulating DOM via JS is deceptively hard to do. I also need to brush up on it. But I think you are on the right track here. Build each HTML element then place all those elements into the main HTML body tag.

Reply:
Agreed. It gets pretty messy quickly. 

One note with the for loop. Since the .json is an object. I don't think it has a length property. You would have to use a for...in loop to go through it. I get what you are trying to do though. Going through the resume.json and get the data from it.